app/controllers/donations.js

```diff
@@ -228,9 +241,14 @@ module.exports = (app) => {
           DonationId: results.createDonation.id,
           amount: amountFloat,
           currency,
-          platformFee: applicationFee,
-          paymentProcessingFee: 0, // TODO: Need to make a separate call for this
-          data: charge,
+          fxCurrency: balanceTransaction.currency,
+          fxAmount: balanceTransaction.amount,
+          fxRate: amountInt/balanceTransaction.amount,
+          fxHostFee: parseInt(balanceTransaction.amount*0.05, 10), // TODO: find a better way than hardcoding
+          fxPlatformFee: fees.applicationFee,
```

Comment:
`fx` stands for "foreign exchange" right? What does a "foreign exchange platform fee" mean?